garak/cli.py

```diff
@@ -327,9 +333,6 @@ def main(arguments=[]) -> None:
     import garak.evaluators
 
     try:
-        if not args.version and not args.report:
-            command.start_run()
-
```

Comment:
talk me through this :)

Reply:
This call has simply moved to after configuration validation is complete to only start in `interactive` mode or when the processing has verified a `run` will occur.